instances/atlas/parcellationEntity/JBA/JBA_overall.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_overall",
+    "@type": "https://openminds.ebrains.eu/sands/ParcellationEntity",
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/JBA_overall"
```

Comment:
cannot be the parent of itself. if it is the highest parent it should just have no parent

Reply:
related to https://github.com/HumanBrainProject/openMINDS_SANDS/pull/151#discussion_r765961546